webservices/schemas.py

```diff
@@ -394,7 +394,9 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'], schemas['Ca
         'ending_image_number': ma.fields.Str(),
         'pdf_url': ma.fields.Str(),
         'is_amended': ma.fields.Boolean(),
+        'document_description': ma.fields.Str(),
     },
```

Comment:
No worries, I was trying to make `document_discription`, and I needed to make `report_type_full` for that